README.md

```diff
@@ -1,66 +1,4 @@
-![cf](https://i.imgur.com/7v5ASc8.png) lab 13 single resource express/mongo api
-======
+READ ME
 
```

Comment:
Try to get in the habit of using markdown format when writing your read me. Also would be a good idea to include instructions on how to "get started".